### PR TITLE
v1.29: kubeadm skew policy for kubelet is n-3

### PR DESCRIPTION
**Pull request by pacoxu**
See https://github.com/kubernetes/kubeadm/pull/2950 and https://github.com/kubernetes/kubeadm/issues/2924

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -559,7 +559,8 @@ version as kubeadm or one version older.
 
 Example:
 * kubeadm is at {{< skew currentVersion >}}
-* kubelet on the host must be at {{< skew currentVersion >}} or {{< skew currentVersionAddMinor -1 >}}
+
+- kubelet on the host must be at {{< skew currentVersion >}}, {{< skew currentVersionAddMinor -1  >}} , {{< skew currentVersionAddMinor -2  >}}  or {{< skew currentVersionAddMinor -3 >}}
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:563]:
```suggestion
- kubelet on the host must be at {{< skew currentVersion >}}, {{< skew currentVersionAddMinor -1 >}}, {{< skew currentVersionAddMinor -2 >}} or {{< skew currentVersionAddMinor -3 >}}
```

Comment (pacoxu) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md]: Updated.

File: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -559,7 +559,8 @@ version as kubeadm or one version older.
 
 Example:
 * kubeadm is at {{< skew currentVersion >}}
-* kubelet on the host must be at {{< skew currentVersion >}} or {{< skew currentVersionAddMinor -1 >}}
+
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:562]: no need for this extra line...

Comment (pacoxu) [reply content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md]: Accidentally added. Removed.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**